windows_containers/enabling-windows-container-workloads.adoc

```diff
@@ -26,6 +26,11 @@ Dual NIC is not supported on WMCO-managed Windows instances.
 
 * You are running an {product-title} cluster version 4.6.8 or later.
 
+[NOTE]
+====
+The WMCO is not supported in clusters that use a xref:../networking/enable-cluster-wide-proxy.html[cluster-wide proxy] because the WMCO is not able to route traffic through the proxy connection for the workloads.
```

Comment:
@mburke5678 Looks like the xref here uses the `.html` extension (should be `.adoc`) and is missing the #anchor. Can you submit another PR to fix this one up please?

FYI @aireilly heads up too

Reply:
ack. Thanks @bergerhoffer